frappe_private_comment/public/js/controls/replies.js

```diff
@@ -0,0 +1,290 @@
+function add_reply_button(time_line_item) {
+  // if reply button already exists, return
+  if ($(time_line_item).find(".custom-actions .reply-btn").length) {
+    return;
+  }
+  const replyButton = $("<button>")
+    .addClass("btn btn-xs btn-link reply-btn")
+    .css({ "font-size": "12px" })
+    .html('<i class="fa fa-reply"></i> Reply')
+    .on("click", () => handle_reply(time_line_item));
+
+  $(time_line_item).find(".custom-actions").append(replyButton);
+}
+
+function render_replies(commentSelector, replies) {
+  const $comment = $(commentSelector);
+  const $prevContainer = $comment.next(".threaded-reply-container");
+  $prevContainer && $prevContainer.remove();
+
+  const $replyContainer = $(
+    '<div class="threaded-reply-container " style="margin-left: 90px; position: relative;"></div>'
```

Comment:
Please do not use inline style unless it is necessary

Reply:
Update: will separate in a different css file (after existing bugs are fixed)